VTEX - Logistics API.json

```diff
@@ -10442,539 +10343,140 @@
+                        "type": "integer",
+                        "format": "int32"
                     },
-                    "PickupPointItemOptions": {
-                        "type": "string",
-                        "nullable": true
+                    "height": {
+                        "type": "integer",
+                        "format": "int32"
                     },
-                    "CanceledDateUtc": {
-                        "type": "string"
+                    "width": {
+                        "type": "integer",
+                        "format": "int32"
                     },
-                    "AuthorizedDateUtc": {
+                    "length": {
+                        "type": "integer",
+                        "format": "int32"
+                    }
+                },
+                "example": {
+                    "weight": 150,
+                    "height": 1,
+                    "width": 1,
+                    "length": 1
+                }
+            },
+            "Location": {
+                "title": "Location",
+                "required": [
+                    "zipCode",
+                    "country",
+                    "inStore"
+                ],
+                "type": "object",
+                "properties": {
+                    "zipCode": {
                         "type": "string"
                     },
-                    "ConfirmedDateUtc": {
+                    "country": {
                         "type": "string"
                     },
-                    "Errors": {
-                        "type": "array",
-                        "items": {
-                            "type": "string"
+                    "inStore": {
+                        "title": "InStore",
+                        "required": [
+                            "IsCheckedIn",
+                            "StoreId"
+                        ],
+                        "type": "object",
+                        "properties": {
+                            "IsCheckedIn": {
+                                "type": "boolean"
+                            },
+                            "StoreId": {
+                                "type": "string",
+                                "nullable": true
+                            }
                         },
-                        "description": ""
-                    },
-                    "IsSucess": {
+                        "example": {
+                            "IsCheckedIn": false,
+                            "StoreId": null
+                        }
+                    }
+                },
+                "example": {
+                    "zipCode": "22220070",
+                    "country": "BRA",
+                    "inStore": {
+                        "IsCheckedIn": false,
+                        "StoreId": null
+                    }
+                }
+            },
+            "InStore": {
```

Comment:
there's another schema that seems to be exactly the same defined as "inStore". which one is correct?